SubscribeIT/ngDesk
# 357-Days before today list layout bug

Closes #357 

Test Case 1
Name :- list layout creating days before today operator
1. Login to ngdesk.
2. Goto -> modules -> tickets -> layouts -> list layouts -> Click on active tickets for system admin list layout.
3. Click on meet all.
4. Add date created, select days before today operator.
5. Observe the field.
6. Set it some value.
7. Hit save.
8. Observe gettings saved.
9. Observe no error.

Test Case 2
Name :- list layout editing days before today operator
1. Login to ngdesk.
2. Goto -> modules -> tickets -> layouts -> list layouts -> Click on active tickets for system admin list layout.
3. Observe and click on meet any.
4. Add date created, select days before today operator.
5. Observe the field.
6. Set it some value.
7. Hit save.
8. Observe gettings saved.
9. Observe no error.

Screenshots :- 

![Screenshot from 2022-02-04 13-06-53](https://user-images.githubusercontent.com/89504119/152490134-586564a7-a86d-4492-9b44-461025a921b6.png)
![Screenshot from 2022-02-04 13-07-17](https://user-images.githubusercontent.com/89504119/152490137-3d6e22db-be81-449f-aed6-cb403043a9a1.png)
![Screenshot from 2022-02-04 13-07-28](https://user-images.githubusercontent.com/89504119/152490142-25087337-5782-4615-82fb-e5bf2d806f65.png)
